server/argv.js

```diff
@@ -0,0 +1,17 @@
+module.exports = require('yargs')
+    .options({
+        c: {
+            alias: 'config',
+            default: 'configs/config.json',
+            describe: 'Configuration file path',
+            nargs: 1,
+            type: 'string'
+        },
+        s: {
+            alias: 'socket',
```

Comment:
Socket is specified in config file and you can change it there.

Just a suggestion:
  1. Check debian helper which change symlinks. It can change symlinks for directories. Can it handle files?
  2. Add project runner to `bin/{{ProjectName}}`.
  3. Remove `server/argv.js` and run project by this code:
```
./bin/{{ProjectName}} # it'll use `confis/config.js` by default
```
or
```
./bin/{{ProjectName}} --config <custom-path-to-config>
```